### ISSUE TITLE
[BUG] PII Access Request Developer Registration

### ISSUE THREAD

**Issue by void-soul** [amzn/selling-partner-api-docs#1642]
Hello,

For getting shipping address via getOrderAddress api, I updated developer profile.

But I got the following answer,

`We have reviewed the information you provided and determined that you do not meet the requirements for Restricted SP-API.

To protect Amazon Customers, we only consider sharing Customer data (“Restricted access”) with third party Developers that offer features that are materially different from existing applications. Their website should be publicly accessible and allow users to learn more about the applications offered. The Developers that meet these criteria must then go through rigorous security reviews with Amazon.`

I updated our official website and also uploaded our application information in the App Store. Since our application is a C/S architecture, we cannot provide complete user functions through the web page, but a download link is provided on the official website.
Then I re-submitted the developer information, but Amazon did not reply and closed the corresponding case. What should I do next to allow Amazon to review my developer qualifications? thank you very much.

**Comment by void-soul** [comment]
most recent Developer Registration case IDS are as following:
9130168021
9185672561

**Comment by void-soul** [comment]
After I waited for 5 days, the case I used to apply for PII was closed, but my developer registration is under review. What is the reason? what should I do? I feel very confused.
My last company has been engaged in MWS development for 4 years, and it has also passed the audit jointly organized by Amazon and Deloitte.
But now I can't start my work, don't even know what I should do.

**Comment by void-soul** [comment]
anybody?

**Comment by void-soul** [comment]
May I take up some of your time? I have two questions:
1: My developer profile is still not editable, even if all related cases have been closed
2: I found that Amazon's evaluation of our application is not "offer features that are materially different from existing applications", but in fact our application adopts the c/s architecture developed by Electron, and there are some special places, such as sellers can import Amazon's latest According to the template file, it generates a standardized upload template, and can also create an unlimited number of personal dictionaries and so on. In addition, our company's operating personnel and our partners have been using our ERP for a long time. They have become accustomed to our operating methods and operating procedures, and it is difficult to transfer to a new platform. Therefore, I urge you to pay attention to my SP-API access upgrade request, thank you

@jchavest13  @rugved1991  @ShivikaK
